wilkes/roster.json

```diff
@@ -0,0 +1,24 @@
+{
+  "namesake": {
+    "name": "",
+    "imageUrl": "",
+    "materials": [
+      {
+        "type": "",
+        "url": ""
+      }
+    ]
+  },
+  "students": [
+    {
+      "name": "Andrew Upton",
+      "github": "nottnottloop",
+      "materials": [
```

Comment:
Great initiative @nottnottloop ! Later in the course we'll be asking for the materials key to be added - you can leave it in for now if you like but we will get you to update as and when with a link to a portfolio site you've created 😃 
Welcome!